pkg/cortex/cortex.go

```diff
@@ -17,6 +17,19 @@ import (
 
 const (
 	query = `aggregation:giantswarm:cluster_release_version{pipeline="testing"}`
+
+	// Amount of time to look back. The longer the time frame, the slower
+	// and more expensive the query.
+	timeRange = 1 * 24 * time.Hour
+
+	// If a cluster has been last more than this much time ago,
```

Comment:
```suggestion
	// If a cluster has been last seen more than this much time ago,
```

Thanks these comments explaining the querying are very useful.